content/_index.md

```diff
@@ -273,7 +273,7 @@ Here are some that we like:
 
 **Return zero exit code on success, non-zero on failure.**
 Exit codes are how scripts determine whether a program succeeded or failed, so you should report this correctly.
-Map the non-zero exit codes to the most important failure modes.
+Map the non-zero exit codes to the most important failure modes, but avoid exit codes above 127.
```

Comment:
trying to add minimal amount of "why" to address [the concern](https://github.com/cli-guidelines/cli-guidelines/pull/92#issuecomment-1776659409):

```suggestion
Map the non-zero exit codes to the most important failure modes, but avoid exit codes above 127 since the highest bit (corresponding to 128) could be set by a shell if the process was terminated.
```

but really not sure if it should go to that detail here.  May be a reference should be added instead?